src/test_utils/katana/genesis.rs

```diff
@@ -121,16 +121,12 @@ impl<T> KatanaGenesisBuilder<T> {
         self.class_hashes.get("kakarot").cloned().ok_or(eyre!("Missing Kakarot class hash"))
     }
 
-    pub fn contract_account_class_hash(&self) -> Result<FieldElement> {
-        self.class_hashes.get("contract_account").cloned().ok_or(eyre!("Missing contract account class hash"))
+    pub fn account_contract_class_hash(&self) -> Result<FieldElement> {
+        self.class_hashes.get("account_contract").cloned().ok_or(eyre!("Missing account contract class hash"))
     }
 
-    pub fn eoa_class_hash(&self) -> Result<FieldElement> {
-        self.class_hashes.get("externally_owned_account").cloned().ok_or(eyre!("Missing eoa account class hash"))
-    }
-
-    pub fn proxy_class_hash(&self) -> Result<FieldElement> {
-        self.class_hashes.get("proxy").cloned().ok_or(eyre!("Missing proxy class hash"))
+    pub fn uninitialized_account_class_hash(&self) -> Result<FieldElement> {
+        self.class_hashes.get("uninitialized_account").cloned().ok_or(eyre!("Missing uninitialized account class hash"))
```

Comment:
Maybe we should have a proper error type for all these errors no? Don't know if it's really relevant there as it concerns test utils, what do you think?

Reply:
Mmmm don't know if it's really worth it, these are test utils that are only used in binaries to set up genesis. Could actually use ˋok_or_eyre`, missed that here